sympy/ntheory/modular.py

```diff
@@ -54,17 +54,12 @@ def crt(m, v, symmetric=False, check=True):
        >>> [639985 % m for m in [99, 97, 95]]
        [49, 76, 65]
 
-    If the moduli are not co-prime, you may receive an incorrect result
-    if you use ``check=False``:
+    If the moduli are not co-prime, you will receive a ValueError.
 
-       >>> crt([12, 6, 17], [3, 4, 2], check=False)
```

Comment:
please restore these 4 lines to show that, with check False, the wrong result might be obtained